lightstep/rand/locked_rand.go

```diff
@@ -60,6 +60,15 @@ func (lr *LockedRand) Uint64() (n uint64) {
 	return
 }
 
+// TwoUint64 generates 2 random uint64 without locking twice.
+func (lr *LockedRand) TwoUint64()  (n1, n2 uint64) {
+	lr.lk.Lock()
+	n1 = lr.r.Uint64()
+	n2 = lr.r.Uint64()
+	lr.lk.Unlock()
```

Comment:
nit: Both here and above, I this it would be preferred to have the `Unlock()` calls be `defer`-ed.

Reply:
I wanted to keep consistency with the other functions in this file!